docs/en/observability/user-experience.asciidoc

```diff
@@ -0,0 +1,185 @@
+// These need to move into the docs repo
+:user-experience: User Experience
+:user-experience-tab: {user-experience} tab
+
+// At some point we need to add a page that talks about how synthetics and
+// RUM/UE work together. Passive/Active, etc.
+
+[[user-experience]]
+= {user-experience}
+
+{user-experience} provides a way to quantify and analyze the perceived performance of your web application.
+Unlike testing environments, {user-experience} data reflects actual users' real-world experiences.
+With just a few lines of code, you can surface key user experience metrics and investigative tools
```

Comment:
In general, I try to avoid linking away in an intro paragraph -- someone just started reading here, and we don't want to send them elsewhere quite yet. With that being said, you're right, this could be more clear. I'll update the text.